projects/storefrontlib/shared/components/ng-select-a11y/ng-select-a11y.directive.ts

```diff
@@ -39,6 +43,8 @@ export class NgSelectA11yDirective implements AfterViewInit {
   protected translationService = inject(TranslationService);
   protected domSanitizer = inject(DomSanitizer);
   private featureConfigService = inject(FeatureConfigService);
+  private selectComponent = inject(NgSelectComponent);
```

Comment:
Lets make these `protected`. We only make `featureConfigService` private because we don't want to expose it to users.

Reply:
@Zeyber, thanks for pointing it out. I's fixed.